testing/e2e/ntfy/setup.sh

```
@@ -0,0 +1,322 @@
+    echo "  ./setup.sh                  # Start server and wait for readiness"
+    echo "  ./setup.sh start-server     # Start server only"
+    echo "  ./setup.sh stop-server      # Stop server"
+    echo "  ./setup.sh --verbose setup-all  # Run with verbose output"
+}
+
+# Load environment variables from .env file
+load_env_file() {
+    if [[ -f "$ENV_FILE" ]]; then
+        debug "Loading environment variables from ${ENV_FILE}"
+        set -a
+        source "$ENV_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)